crop.module

```diff
@@ -88,3 +88,17 @@ function crop_form_media_bundle_form_alter(&$form, FormStateInterface $form_stat
 function crop_media_bundle_form_builder($entity_type, MediaBundleInterface $bundle, &$form, FormStateInterface $form_state) {
   $bundle->setThirdPartySetting('crop', 'image_field', $form_state->getValue('image_field'));
 }
+
+/**
+ * Returns a list of available crop type names.
+ *
+ * This list can include types that are queued for addition or deletion.
+ *
+ * @return string[]
+ *   An array of crop type labels, keyed by the crop type name.
+ */
+function crop_type_get_names() {
```

Comment:
It feels that having this as a static function on crop type class would be better.